src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java

```diff
@@ -897,6 +1098,31 @@ public SELF containsExactly(double... values) {
     return myself;
   }
 
+  /**
+   * Verifies that the actual group contains only the values of the given array and nothing else, <b>in order</b>.
+   * <p>
+   * Example :
+   * <pre><code class='java'> // assertion will pass
+   * assertThat(new double[] { 1.0, 2.0, 1.0 }).containsExactly(new Double[] {1.0, 2.0, 1.0});
+   *
+   * // assertion will fail as actual and expected order differ
+   * assertThat(new double[] { 1.0, 2.0, 1.0 }).containsExactly(new Double[] {2.0, 1.0, 1.0});</code></pre>
+   *
+   * @param values the given values.
+   * @return {@code this} assertion object.
+   * @throws NullPointerException if the given argument is {@code null}.
+   * @throws AssertionError if the actual group is {@code null}.
+   * @throws AssertionError if the actual group does not contain the given values with same order, i.e. the actual group
+   *           contains some or none of the given values, or the actual group contains more values than the given ones
+   *           or values are the same but the order is not.
+   * @since 3.19.0
+   */
+  public SELF containsExactly(Double[] values) {
```

Comment:
Please also add
```java
public SELF containsExactly(Double[] values, Offset<Double> precision)
```

Reply:
https://github.com/assertj/assertj-core/blob/c0f48d3858e047ed0421a36c435e6f1b5503edcb/src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java#L951

It's already there